static/webxdc-preload.js

```diff
@@ -1,3 +1,47 @@
+/**
+ * @typedef {import('webxdc-types').RealtimeListener} RT
+ * @type {RT}
+ */
+class RealtimeListener {
+  constructor(ipc) {
+    this.listener = null
+    this.trashed = false
+    this.ipc = ipc
```

Comment:
we should not expose that, it might be a security issue, do not store a reference to it in the class.

Reply:
I don't know why, but wrapper functions around IPC can not be transported through the electron portal...
I have to think of something else then..

Reply:
nvm